### DISCUSSION TITLE
Playground params via url

### DISCUSSION THREAD

**Discussion by husayt** [esm-dev/esm.sh#806] [General] [answered]
Playground is very useful and sometimes it would be very useful to share a particular example via url or maybe open playground with some code. Is there a way to do that?

**Comment by ije** [comment, marked as answer]
i am working on a better version that allows you share your code, also going to open the editor code too

**Reply by husayt** [reply]
Already code.sh is a better playground than codepen and others. This is the only thing which is holding it from replacing them.

**Reply by ije** [reply]
thank you for such a high comment 😊
